src/CreepLaw/DiffusionCreep.jl

```diff
@@ -255,36 +259,35 @@ Returns diffusion creep stress as a function of 2nd invariant of the strain rate
 @inline function compute_τII(
     a::DiffusionCreep, EpsII::_T; T=one(precision(a)), P=zero(precision(a)), f=one(precision(a)), d=one(precision(a)), kwargs...
 ) where {_T}
-    @unpack_val r, p, A, E, V, R = a
+    @unpack_val n, r, p, A, E, V, R = a
     FT, FE = a.FT, a.FE
 
     τ =
-        fastpow(A, -1 ) *
-                EpsII * FE *
-        fastpow(f, -r ) *
-        fastpow(d, -p ) *
-        exp((E + P * V) / (R * T)) / FT
+        fastpow(A, -1 / n) *
```

Comment:
Division is 5-6 times slower than multiplication. You could precompute the inverse of `n` and multiply instead, i.e.
`_n = inv(v)` and then do for example `fastpow(A, -p*_n)` or whatever.

Reply:
Done.